oem/ibm/libpldmresponder/file_io_type_pcie.cpp

```diff
@@ -904,12 +919,9 @@ void PCIeInfoHandler::parseTopologyData()
             remote_bottom_port_location.begin(),
             remote_bottom_port_location.end());
 
-        struct SlotLocCode_t* slot_data =
-            (struct SlotLocCode_t*)(((uint8_t*)single_entry_data) +
-                                    htobe16(single_entry_data
-                                                ->slot_loc_codes_offset));
-
-        if (slot_data == nullptr)
+        uint8_t* suffix_data = (uint8_t*)slot_data + 2 +
```

Comment:
can you explain why you made this change in commit message or here as a comment?